### ISSUE TITLE
`NEWS` is required, even if `foreign` supplied to automake

### ISSUE THREAD

**Issue by rrthomas** [gvvaughan/slingshot#12]
The rules in `release.mk` require `NEWS` even though bootstrap doesn't insist on it if `foreign` is passed to automake. Without the file, the make system gets into a loop.

**Comment by gvvaughan** [comment]
Yes, the rules are based on gnulib's `GNUmakefile` and/or `maint.mk`.  And it is a GNU standard that all projects should have a `NEWS` file.  I've tried to relax things by automatically soft-linking to `NEWS.md`, which I've been using everywhere.  Keeping `NEWS.md` in GHFMD means generated release announcements (from `make announce` or `make stable` etc) can be pasted into the GitHub release announcements.

Looking at the rules, I can see how having no `NEWS` as well as no `NEWS.md` would cause problems.  Is that really what you want?  I'd suggest adding a `bootstrap.conf` hook-function to create an empty NEWS file before running autoconf et. al. (and as long as you have `foreign` set, you can remove it afterwards in another cleanup hook).

It's annoying that the GNU tools want either foreign setting or a `NEWS` file spelled exactly that way...  I want to finish prototyping Luke to be free from that tyranny among others.

**Comment by gvvaughan** [comment]
Correction.  Slingshot requires `NEWS.md` for the release rules to work, as it tracks changes since the last release, ensures everything inside is properly updated before a release by maintaining a NEWS-hash checksum, and then extracts the relevant parts into the announcement message for pasting into an announcement email and/or github release message.

The `NEWS` link is still an ugly wart enforced by Automake though.

**Comment by rrthomas** [comment]
Yes, I believe that for this project, no `NEWS` is good `NEWS`. But since it's just my private ad-hoc libraries, I'm happy with a workaround until I have Luke or whatever.
